packages/fuels/src/cli/commands/dev/autoStartFuelCore.test.ts

```diff
@@ -1,39 +1,101 @@
+import * as testUtilsMod from '@fuel-ts/wallet/test-utils';
+import { existsSync, rmSync } from 'fs';
+import { join } from 'path';
+
 import { fuelsConfig } from '../../../../test/fixtures/fuels.config';
+import type { FuelsConfig } from '../../types';
 
+import type { FuelCoreNode } from './autoStartFuelCore';
 import { autoStartFuelCore } from './autoStartFuelCore';
-import * as startFuelCoreMod from './startFuelCore';
+
+vi.mock('@fuel-ts/wallet/test-utils', async () => {
+  const mod = await vi.importActual('@fuel-ts/wallet/test-utils');
+  return {
+    __esModule: true,
+    ...mod,
+  };
+});
 
 /**
  * @group node
  */
 describe('autoStartFuelCore', () => {
-  function mockStartFuelCore() {
-    const startFuelCore = vi
-      .spyOn(startFuelCoreMod, 'startFuelCore')
-      .mockResolvedValue({} as unknown as startFuelCoreMod.FuelCoreNode);
+  afterEach(() => {
+    vi.restoreAllMocks();
+
+    const chainConfig = join(fuelsConfig.basePath, '.fuels', 'chainConfig.json');
+    if (existsSync(chainConfig)) {
+      rmSync(chainConfig);
+    }
+  });
 
-    return { startFuelCore };
+  function mockLaunchNode() {
+    vi.spyOn(testUtilsMod, 'launchNode').mockImplementation((..._: unknown[]) =>
+      Promise.resolve({ cleanup: () => {}, ip: '0.0.0.0', port: '4000' })
+    );
```

Comment:
It would be better to return a named variable with the spy instance to keep consistency with other mocking strategies across the codebase. Also, if you don't need to modify the implementation but rather return something else, you can use `mockReturnValue`.

```suggestion
    const cleanup = Promise.resolve({ cleanup: () => {}, ip: '0.0.0.0', port: '4000' });
    const launchNode = vi.spyOn(testUtilsMod, 'launchNode').mockReturnValue(cleanup);
    return { launchNode };
```

Then, on the test cases using it:


```ts
const { launchNode } = mockLaunchNode();

// ...

expect(launchNode).toHaveBeenCalledTimes(0);
```